src/main/java/io/leonis/torch/component/ZosmaBackground.java

```diff
@@ -0,0 +1,51 @@
+package io.leonis.torch.component;
+
+import com.googlecode.lanterna.*;
+import com.googlecode.lanterna.gui2.*;
+import lombok.Value;
+
+import java.awt.Color;
+
+/**
+ *
+ * @author jeroen.dejong.
+ */
+@Value
```

Comment:
@Value adds nothing here besides making the field `text` final. please remove `@Value` in favor of an explicit `final`